ChiModules/KEigenvalueSolver/IterativeOperations/set_k_source.cc

```diff
@@ -80,36 +74,36 @@ void KEigenvalue::Solver::SetKSource(LBSGroupset& groupset,
       {
         unsigned int ell = m_to_ell_em_map[m].ell;
 
-        int64_t ir = full_cell_view.MapDOF(i,m,0);
-        q_mom      = &q_moments_local[ir];
+        int64_t ir        = full_cell_view.MapDOF(i,m,0);
+        double* q_mom     = &q_moments_local[ir];
 
-        phi_oldp  = &phi_old_local[ir];
-        phi_prevp = &phi_prev_local[ir];
+        double* phi_oldp  = &phi_old_local[ir];
+        double* phi_prevp = &phi_prev_local[ir];
 
         // ----- Loop over groupset groups
         for (int g=gs_i; g<=gs_f; g++)
         {
           // ----- Contribute scattering
-          inscat_g = 0.0;
-          if ((ell < xs->transfer_matrix.size()) && (!suppress_phi_old) )
+          double inscat_g = 0.0;
+          if ((ell < xs->transfer_matrix.size()) && (apply_scatter_src) )
```

Comment:
spacing and parentheses is not consistent.

Reply:
Corrected.